src/beacons/use-maybe-show-share-travel-habits-screen.tsx

```diff
@@ -1,22 +1,22 @@
-import {useCallback, useEffect, useRef} from 'react';
+import {useCallback, useEffect, useRef, useState} from 'react';
 import {storage} from '@atb/storage';
 
 import {useRemoteConfig} from '@atb/RemoteConfigContext';
 import {useAppStateStatus} from '@atb/utils/use-app-state-status';
 import {useHasSeenShareTravelHabitsScreen} from './use-has-seen-share-travel-habits-screen';
 import {useAppState} from '@atb/AppContext';
+
 import {useBeaconsState} from './BeaconsContext';
 
 export const shareTravelHabitsSessionCountKey =
   '@ATB_share_travel_habits_session_count';
 
-export const useMaybeShowShareTravelHabitsScreen = (
-  showShareTravelHabitsScreen: () => void,
-) => {
+export const useShouldShowShareTravelHabitsScreen = () => {
```

Comment:
Note: I'll rename this file after review (so that the file doesn't show as deleted and completely new while reviewing)